lib/modules/manager/devbox/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { Category } from '../../../constants';
+import { NixhubDatasource } from '../../datasource/nixhub';
+import { NodeVersionDatasource } from '../../datasource/node-version';
+
+export { extractPackageFile } from './extract';
+export { updateArtifacts } from './artifacts';
+
+export const supportsLockFileMaintenance = false; // not yet
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)devbox\\.json$'],
+  commitMessageTopic: 'devbox/{{depName}}',
+  commitMessageExtra: 'to {{newValue}}',
```

Comment:
Are you sure you want to change the default for this? It can be very handy to have a different format for major and non-major updates